lib/membrane/video_compositor/wgpu/wgpu_adapter.ex

```diff
@@ -128,10 +128,10 @@ defmodule Membrane.VideoCompositor.WgpuAdapter do
         ) :: :ok | {:error, :bad_video_index}
   def update_transformations(state, id, transformations) do
     case Native.update_transformations(state, id, transformations) do
-      :ok ->
+      {:ok, :ok} ->
```

Comment:
I refuse to merge this before I take a stab at fixing it

Reply:
GL HF